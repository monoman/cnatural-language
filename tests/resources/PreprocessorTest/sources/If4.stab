#if XXX == false && !(XXX == true) // comment
// Section
#endif
