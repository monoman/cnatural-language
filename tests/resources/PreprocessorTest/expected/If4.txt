#if expression: true, skipped: false
....----
....// Section
....----
#end
